presidio-analyzer/presidio_analyzer/predefined_recognizers/in_passport_recognizer.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional, List, Tuple
+
+from presidio_analyzer import Pattern, PatternRecognizer
+
+class InPassportRecognizer(PatternRecognizer):
+    """
+    Recognizes Indian Passport Number.
+
+    Indian Passport Number is a eight digit alphanumeric number.
+
+    :param patterns: List of patterns to be used by this recognizer
+    :param context: List of context words to increase confidence in detection
+    :param supported_language: Language this recognizer supports
+    :param supported_entity: The entity this recognizer can detect
+    
+    """
+
+    PATTERNS = [
+        Pattern(
+            "PASSPORT",
+            r"\b[A-Z][1-9][0-9]{2}[0-9]{4}[1-9]\b",
+            0.7,
```

Comment:
Please reduce the confidence to 0.1 to avoid false positives. Score can go up with context words.

Reply:
Done